docs/tutorials/ocp-ibm.md

```diff
@@ -0,0 +1,82 @@
+
+# Red Hat OpenShift on IBM Cloud
+
+The [quick start guide](https://admiralty.io/docs/quick_start) provides clear instructions how to use Admiralty on Kubernetes clusters. The only
+thing you need to pay special attention to is how to create a kubeconfig secret that would work in your OpenShift cluster on IBM Cloud. This tutorial will
+guide you how to create the kubeconfig secret when you use the Red Hat OpenShift on IBM Cloud service.
+
+## Prerequisites
+- the [Red Hat OpenShift on IBM Cloud service](https://www.ibm.com/cloud/openshift) 
+- the required [CLI tools](https://cloud.ibm.com/docs/openshift?topic=openshift-openshift-cli) ( e.g., IBMCLOUD CLI and OpenShift CLI (oc) )
+
+## Kubeconfig for Authentication
+You can follow this [link](https://cloud.ibm.com/docs/openshift?topic=openshift-access_cluster) to access your OpenShift cluster.
+After you connect to the OpenShift cluster, you can use the following IBMCLOUD CLI command
+to retrieve the kubeconfig file.
+```bash
+export KUBECONFIG=~/.kube/config
+
+ibmcloud oc cluster config --cluster <your cluster name> --admin
+```
+Your config file should look like the following:
+``` 
+apiVersion: v1
+clusters:
+- cluster:
+    server: <api server>
+  name: <cluster name>
+contexts:
+- context:
+    cluster: <cluster name>
+    namespace: default
+    user: <admin user name>
+  name: <context name>
+current-context: <context name>
+kind: Config
+preferences: {}
+users:
+- name: <admin user name>
+  user:
+    client-certificate: <path to client certificate>
+    client-key: <path to client key>
+```
+Let's modify the config file to the following format below:
+```
+apiVersion: v1
+clusters:
+- cluster:
+    server: <api server>
+    certificate-authority-data: <ca data>
+  name: <cluster name>
+contexts:
+- context:
+    cluster: <cluster name>
+    namespace: default
+    user: <admin user name>
+  name: <context name>
+current-context: <context name>
+kind: Config
+preferences: {}
+users:
+- name: <admin user name>
+  user:
+    token: <service account token>
+```
+The fields, client-certificate and client-key, are being removed and certificate-authority-data and token fields are added.

```

Comment:
Sure. I can provide the jq command. Shall I create another PR ?

Reply:
No, please update this PR to use jq.

Reply:
done.

Reply:
You now get the config twice, and the CA cert twice... this could be simplified.

Actually, I'm thinking this should become part of the quick start page itself. Using [Tabs/TabItem](https://docusaurus.io/docs/markdown-features/tabs), the user could select between kind and Red Hat OpenShift on IBM Cloud. (We'd add GKE/EKS/AKS tabs too later.) What do you think?